models/i18n.cfc

```diff
@@ -35,15 +38,34 @@ component singleton accessors="true" {
 	}
 
 	/**
-	 * Reads,parses,saves the locale and resource bundles defined in the config. Called only internally by the framework. Use at your own risk
+	 * Reads,parses,saves the locale and resource bundles defined in the config.
+	 *
+	 * @throws i18N.DefaultSettingsInvalidException
 	 */
-	void function configure() {
+	void function onDIComplete() {
 		// Default instance settings
-		variables.localeStorage         = variables.controller.getSetting( "LocaleStorage" );
-		variables.defaultResourceBundle = variables.controller.getSetting( "DefaultResourceBundle" );
-		variables.defaultLocale         = variables.controller.getSetting( "DefaultLocale" );
-
-		// set locale setup on configuration file
+		variables.localeStorage         = variables.settings.LocaleStorage;
+		variables.defaultResourceBundle = variables.settings.DefaultResourceBundle;
+		variables.defaultLocale         = variables.settings.DefaultLocale;
+
+		// instantiate storage service for locale storage
+		//		try {
+		variables.storageService = wirebox.getInstance(
```

Comment:
I don't think it's necessary for you to send in the `initArguments` to get the right storage object.  This should be done automatically by cbStorages.  All we need to do, is request the right storage object according to the `localeStorage` setting.

Reply:
Yes I see now. A bit hidden in the annotations of the storages init functions :-)